### ISSUE TITLE
Add aria-label to each button

### ISSUE THREAD

**Issue by brandonbyr4** [brandonbyr4/comicfun#27]
Each button in the /index.js and /tool.js should have an appropriate aria-label prop.

**Comment by Rishav1707** [comment]
Shall I work on this issue? @brandonbyr4
